README.md

```diff
@@ -4,7 +4,43 @@ Parsy is an abbreviation of `Polder Knowledge Pattern Library`. Combined with th
 latin word for component `pars`. This repository contains the definitions of the 
 patterns that we have approved.
 
-## Create a proposal
+## What is a pattern
+
+In Parsy terms a pattern is a filed template. Just a page of documentation how a common UX/UI
+problem should be solved. We believe that a basic pattern is very small. The more complex patterns
+are always a composite of smaller patterns. In extreme you can state that a html element is a kind of
+pattern to solve a common UI problem.
+
+A pattern doesn't need to have a style. In some situations it might be useful to have default colors
+and font sizes but it is up to the user of the pattern to decide if that should be changed or not. Inspired
+by [empties][empties-bourbon] by bourbon.
+
+A pattern can have one or multiple implementations. We believe that a pattern should solve a problem without forcing
+you to use a certain framework. Therefor a pattern starts with a good description of the goal of the pattern. If this
+ solves your problem you can either use an existing implementation or create your own. In some eco-systems it is not 
+ easy to adopt a new framework or library. You should be able to choose!
+ 
+## Why?
+
+We are sure that we are solving the same UX/UI problems every day. Sometimes the basic patterns and sometimes the 
```

Comment:
`Sometimes these problems relate to basic patterns and other times they are more complex`